### ISSUE TITLE
Incorrect subdomain resolving in Dev env.

### ISSUE THREAD

**Issue by manoharkshetty** [solidusio/solidus_multi_domain#26]
When I am working on Dev env, the subdomain resolving is incorrect.

Store info:
Name: `store1`, Code: `store1`, url: `store1.lvh.me:3000` ( i.e `store1.localhost:3000` )

But in the logs I see that the gem is looking for the code `store1.lvh.me` (or `store1.localhost` if I try `store1.localhost:3000` )

```
Spree::Store Load (0.2ms)  SELECT  "spree_stores".* FROM "spree_stores" WHERE "spree_stores"."code" = $1 LIMIT 1  [["code", "store1.lvh.me"]]
```

Is it a bug or Am I doing something wrong?

Thanks in advance.

gem version 1.1.2

**Comment by jgujgu** [comment]
I have the URL stored as `http://store1.lvh.me:3000/`. I don't know if this will solve your problem.

**Comment by adammathys** [comment]
I believe this issue is solved by #41.
